src/services/zapper.ts

```diff
@@ -304,7 +306,8 @@ export class ZapperService extends Service {
       slippagePercentage: slippagePercentage.toString(),
       api_key: this.ctx.zapper,
       shouldSellEntireBalance: "true",
-      skipGasEstimate: skipGasEstimate ? "true" : "false"
+      skipGasEstimate: skipGasEstimate ? "true" : "false",
+      ...(signature && { signature })
```

Comment:
just using `signature` here would be fine

Reply:
it should not

Reply:
Will leave it as it is since it errors out otherwise